utils/psh-applets/echo.md

```diff
@@ -0,0 +1,33 @@
+# echo
+
+The `echo` utility provided as a `psh` applet can be used to print a line of text to standard output.
+
+---
+
+If used with `-h` parameter it prints the help message with possible arguments and parameters as follows:
+
+```bash
+Usage: %s [options] [files]
```

Comment:
`[string]`? or maybe `[string1] ...`?

Not sure about `%s`, probably we should put `echo` here :)